src/Microsoft.Azure.WebJobs.Host/FuncConverter.cs

```diff
@@ -3,19 +3,21 @@
 
 using System;
 using Microsoft.Azure.WebJobs.Host.Bindings;
+using System.Threading.Tasks;
 
 namespace Microsoft.Azure.WebJobs
```

Comment:
>WebJobs [](start = 26, length = 7)

It looked a lot cleaner to specifically call out delegates rather than use Func<> everywhere.